kustomize/components/clair/clair.deployment.yaml

```diff
@@ -45,8 +45,9 @@ spec:
             tcpSocket:
               port: 8080
           livelinessProbe:
-            tcpSocket:
-              port: 8080
+            httpGet:
```

Comment:
Can we change the `startupProbe` and `readinessProbe` to also use this port?

Reply:
I don't think that'd make sense, as the the `readinessProbe` is "ready to serve traffic", which it doesn't do on this port. The `startupProbe` could be moved over.

Reply:
Fixing the `startupProbe` in this way won't actually fix the problem of blocking the Quay deployment on Clair, because the `config-tool` will attempt to ping the Clair host as part of its validation step. If Clair refuses the connection (which it does currently), then the validation will fail and the Quay pod will not start.

Reply:
Okay, well, this change encodes liveness and readiness for the pod more accurately. If `config-tool` wants to not wait for clair to be ready, that's a different change.